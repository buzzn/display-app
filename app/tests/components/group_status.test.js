import React from 'react';
import renderer from 'react-test-renderer';
import GroupStatus from '../../components/group_status';

describe('group status component tests', () => {
  test('renders n.a. value correctly', () => {
    const tree = renderer.create(
      <GroupStatus text="text"/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('renders value correctly', () => {
    const tree = renderer.create(
      <GroupStatus text="text" value={ 500 }/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
