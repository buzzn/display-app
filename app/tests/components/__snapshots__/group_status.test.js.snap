// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`group status component tests renders n.a. value correctly 1`] = `
<div
  style={
    Object {
      "left": "30%",
      "position": "absolute",
      "top": "40px",
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "#e0e0e0",
        "borderRadius": "10px",
        "height": "184px",
        "margin": "auto",
        "width": "184px",
      }
    }
  >
    <div
      style={
        Object {
          "fontSize": "70px",
          "paddingTop": "20px",
          "textAlign": "center",
        }
      }
    >
      n.a
    </div>
    <div
      style={
        Object {
          "fontSize": "20px",
          "textAlign": "center",
        }
      }
    >
      text
    </div>
  </div>
</div>
`;

exports[`group status component tests renders value correctly 1`] = `
<div
  style={
    Object {
      "left": "30%",
      "position": "absolute",
      "top": "40px",
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "#e0e0e0",
        "borderRadius": "10px",
        "height": "184px",
        "margin": "auto",
        "width": "184px",
      }
    }
  >
    <div
      style={
        Object {
          "fontSize": "70px",
          "paddingTop": "20px",
          "textAlign": "center",
        }
      }
    >
      50000%
    </div>
    <div
      style={
        Object {
          "fontSize": "20px",
          "textAlign": "center",
        }
      }
    >
      text
    </div>
  </div>
</div>
`;
